### PR TITLE
Allow to use tool on Node 6

### PR DESCRIPTION
**Pull request by ai**
Because of a few political problems I need to support Node.js 6 in Autoprefixer for few months.

Estimo uses Node.js 8+ code in `chromeDetection.js` and it [breaks CI](https://travis-ci.org/postcss/autoprefixer/jobs/568105873).

I suggest adding `ESTIMO_DISABLE` environment variable to not even loading `chromeDetection.js` in `findChrome.js`. It could be useful in many cases.

I didn’t move `PUPPETEER_SKIP_CHROMIUM_DOWNLOAD`, because with `PUPPETEER_SKIP_CHROMIUM_DOWNLOAD` Estimo looks for local Chrome now (and I do not want breaking changes).

### CODE REVIEW

File: chrome.json [hunk 1]
```diff
@@ -1 +1 @@
-{ "executablePath": "" }
\ No newline at end of file
+{"executablePath":"/usr/bin/google-chrome-stable"}
```
Comment (mbalabash) [inline chrome.json]: I think you accidentally add this line. `executablePath` should be empty by default.

File: scripts/findChrome.js [hunk 2]
```diff
@@ -1,3 +1,5 @@
-const { findChrome} = require('./chromeDetection')
+if (!process.env.ESTIMO_DISABLE) process.exit()
```
Comment (mbalabash) [inline scripts/findChrome.js]: If ESTIMO_DISABLE is true you don't want to execute findChrome. Am i right?
If so, why you are using here negative operator?

**Comment by ai** [comment]
@mbalabash oops. Thanks for the review. I fixed changes.

**Comment by mbalabash** [comment]
Released in estimo@1.1.5

**Pull request merged**